tests/helpers/aptible-helpers.js

```diff
@@ -449,5 +435,7 @@ Ember.Test.registerHelper('stubUser', function(app, userData={}){
 });
 
 Ember.Test.registerAsyncHelper('triggerSlider', function(app, selector, argument){
-  Ember.$(selector).trigger('slide', argument);
+  let slider = findWithAssert(selector);
+  slider.trigger('slide', argument);
+  slider.trigger('set', argument);
```

Comment:
appears to be no harm in trigger set & slide together, and the 'set' is necessary for the `service-scaler` component to show its 'scale' button